frontend/src/components/common/table/table.tsx

```diff
@@ -0,0 +1,61 @@
+import { Column, useTable } from 'react-table';
+import { FC } from 'common/types/types';
+import styles from './table/styles.module.scss';
+
+type Props = {
+  columns: Column[];
+  data: unknown[];
+};
+
+const Table: FC<Props> = ({ columns, data }) => {
+  const tableInstance = useTable({
+    columns: columns as Column<Record<string, string>>[],
+    data: data as Record<string, string>[],
+  });
```

Comment:
Can't we assign appropriate types in props?

```suggestion
type Props = {
  columns: Column<Record<string, string>>[];
  data: Record<string, string>[];
};

const Table: FC<Props> = ({ columns, data }) => {
  const tableInstance = useTable({
    columns,
    data,
  });
```

Reply:
no. data should be in unknown state